src/domain/entities/DataSetList.ts

```diff
@@ -0,0 +1,24 @@
+import { DataSet } from "$/domain/entities/DataSet";
+import { User } from "$/domain/entities/User";
+import { Struct } from "$/domain/entities/generic/Struct";
+
+export type DataSetListAttrs = Pick<
+    DataSet,
+    "id" | "name" | "lastUpdated" | "permissions" | "canBeUpdated"
+>;
+
+export class DataSetList extends Struct<DataSetListAttrs>() {
```

Comment:
Jorge does not recommend creating separate entities by complexity, but let's keep it for simplicity.